contracts/account/modules/ERC7579DelayedExecutor.sol

```diff
@@ -273,7 +273,7 @@ abstract contract ERC7579DelayedExecutor is ERC7579Executor {
         bytes32 /* salt */,
         bytes32 /* mode */,
         bytes calldata data
-    ) internal view virtual override returns (bool valid, bytes calldata executionCalldata) {
+    ) internal virtual override returns (bool valid, bytes calldata executionCalldata) {
```

Comment:
I realized it is a valid use case to consume a nonce here. In the case a contract extends and perform other storage updates during cancellation, they should make their signatures for cancellations replayable to avoid unexpected side effects of recancelling.

Anyway, it's a long discussion and I think it's just reasonable to remove the `view` modifier in the `_validate*` functions of the ERC7579Executor and ERC7579DelayedExecutor

Reply:
Also just realized it's unnecessary to return a `boolean valid` then. That pattern applies when the function is view.

Reply:
Opened #158 to solve these first